.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.js]
+indent_style = space
+indent_size = 2
+
```

Comment:
Not sure editorconfig needs to be part of this PR, (or needed at all) - using `standard` to enforce linting.

Reply:
If you want to make it a separate PR that's fine, but I'd prefer for it to not be part of this PR - apologies for the bureaucracy.